src/osp/OSPApplication.cpp

```diff
@@ -30,3 +30,13 @@ OSPApplication::OSPApplication()
 {
 
 }
+
+void OSPApplication::debug_add_package(Package&& p)
+{
+    m_packages.emplace(p.get_prefix(), std::move(p));
+}
+
+Package& OSPApplication::debug_get_package(resprefix_t prefix)
```

Comment:
resprefix_t is typedefed as std::string

but here you're taking it by value for use with a function that doesn't require ownership semantics.

It's far far more likely that what you want to do here is take an std::string_view (Give it an appropriate typedef if you want), and then use std::map::find(), and if the result of find() == std::map::end(), then throw the exception that std::map::at() would throw.

If you don't want to use string_view (I strongly recommend it for this situation), then at the least, take the argument as const&